### PR TITLE
API v2 Storefront OpenAPI docs fixes

### PR DESCRIPTION
**Pull request by damianlegawiec**

### CODE REVIEW

File: api/docs/v2/storefront/index.yaml [hunk 1]
```diff
@@ -285,13 +285,13 @@ paths:
         - Checkout
       responses:
         '200':
-          description: 200 response means that the Checkout was transitioned to the next step
+          description: Checkout was advanced to to the furthest step
```
Comment (kgorazd) [inline api/docs/v2/storefront/index.yaml]: double `to`

File: api/docs/v2/storefront/index.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ info:
       Please read our introduction to the API v2 [insert link here]
     </p>
     <p>
-      Click here to download Postman collection with example operations
+      <a href="https://raw.githubusercontent.com/spree/spree/master/api/docs/v2/storefront/index.yaml">Import this documentation to Postman</a>
```
Comment (piotrleniec-spark) [inline api/docs/v2/storefront/index.yaml]: Maybe we could add `target="_blank" rel="noopener"` attributes? It'd make browsing docs more convenient.

File: api/docs/v2/storefront/index.yaml [hunk 3]
```diff
@@ -1591,6 +1666,27 @@ components:
...
+          Format:
+          <ul>
+            <li>parent,taxonomy,children,image,products</li>
+            <li>children.children</li>
+            <li>parent.children</li>
+            <li>children,products.default_variant</li>
+          </ul>
+        <p>
+          More information:
+          <a href="https://jsonapi.org/format/#fetching-includes">https://jsonapi.org/format/#fetching-includes</a>
```
Comment (piotrleniec-spark) [inline api/docs/v2/storefront/index.yaml]: Maybe we could add `target="_blank" rel="noopener"` attributes? It'd make browsing docs more convenient.

**Review by kgorazd — APPROVED** [review]

File: core/spec/models/spree/taxon_image_spec.rb [hunk 4]
```diff
@@ -6,6 +6,24 @@
     let(:image_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'thinking-cat.jpg') }
     let(:text_file) { File.open(Spree::Core::Engine.root + 'spec/fixtures' + 'text-file.txt') }
 
+    it 'has attachment present' do
```
Comment (piotrleniec-spark) [inline core/spec/models/spree/taxon_image_spec.rb]: Right now, these tests are dependant on `Rails.application.config.use_paperclip` setting and only one test branch will be covered. Maybe we could set the value and test multiple conditions?

File: api/docs/v2/storefront/index.yaml [hunk 5]
```diff
@@ -194,13 +194,13 @@ paths:
         - Cart
       responses:
         '200':
-          description: 200 response means that coupon code was applied successfully
+          description: Cupon code was applied successfully
```
Comment (piotrleniec-spark) [inline api/docs/v2/storefront/index.yaml]: `Coupon`

**Review by pl-spark — CHANGES_REQUESTED** [review]

File: core/spec/models/spree/taxon_image_spec.rb (hunk 4 above)
Comment (damianlegawiec) [reply core/spec/models/spree/taxon_image_spec.rb]: @piotrleniec-spark both active storage and paperclip are tested inside those tests. All CI builds are tested against AS and paperclip as a separate processes

**Review by pl-spark — APPROVED** [review]

**Pull request merged**